_cucumber/Dockerfile

```diff
@@ -4,8 +4,16 @@ RUN groupadd -g 1000 testing
 RUN useradd -g testing -m -s /bin/bash -u 1000 testing
 USER testing
 WORKDIR /home/testing
+
+USER root
```

Comment:
@robpblake I knew you'd say that :)

I tried using `USER testing` originally but I got an error:

`mv: cannot move '/tmp/phantomjs/bin/phantomjs' to '/usr/bin/phantomjs': Permission denied`